bot/cogs/snakes.py

```diff
@@ -28,6 +39,17 @@ def __init__(self, bot: AutoShardedBot):
         :param name: Optional, the name of the snake to get information for - omit for a random snake
         :return: A dict containing information on a snake
         """
+        if name:
+            name = name.lower()
+            if name == "python":
+                return ("python", self.python_info, self.python_image)
+            else:
+                for key in self.db.keys():
+                    if key.lower() == name.lower():
+                        return (key, self.db[key][0], self.db[key][1])
+        else:
+            key = random.choice(list(self.db.keys()))
```

Comment:
You don't need the `.keys()`. casting a dict to list will already return the keys.

```py
key = random.choice(list(self.db))
```